src/test/resources/movie-test.md

```diff
@@ -358,4 +358,98 @@ mutation {
 MERGE (actor:Actor {userId:$actorUserId}) SET actor.name=$actorName
 WITH actor
 RETURN actor { .name } AS actor
+```
+
+### Should auto generate `add` relationship mutations for arrays
+
+```graphql
+mutation {
+  add: addMovieGenres(movieId: 1, genres: ["Action", "Fantasy"]) {
+    title
+  }
+}
+```
+
+```params
+{"movieId": 1, "genres": ["Action", "Fantasy"]}
+```
+
+```cypher
+MATCH (movie:Movie {movieId:$movieId})
+WITH movie
+UNWIND $genres as genreID
+MATCH (genre:Genre {name:genreID})
+MERGE (movie)-[:GENRE]->(genre)
+WITH movie
```

Comment:
this doesn't work, we'd need to use either `WITH DISTINCT movie` 
or a FOREACH instead of the UNWIND (which wouldn't work as you cannot MATCH inside of FOREACH